docs/groovy-script/mods/selectionguicrafting/category.md

```diff
@@ -0,0 +1,187 @@
+    ```
+
+- `GsEnum.QueueType`. If the recipes in this category can be queued. Can be overridden by the recipe. Allowed values are: `true`, `false` or `'YES'`, `'NO'`. (Default `'YES'`).
+
+    ```groovy:no-line-numbers
+    queueType(String)
+    queueType(boolean)
+    queueType(GsEnum.QueueType)
+    ```
+
+- `GsEnum.SoundType`. How the sounds will be played. Can be overridden by the recipe. Allowed values are: `'RANDOM'` or `'COMBINED'`. (Default `'RANDOM'`).
+
+    ```groovy:no-line-numbers
+    soundType(String)
+    soundType(GsEnum.SoundType)
+    ```
+
+- `ResourceLocation`. The path to the background texture. The resource must be loaded via external methods. [Textures that come with the mod](https://github.com/Ender-Development/selection-gui-crafting-continued/tree/master/src/main/resources/assets/selectionguicrafting/textures/gui/background). (Default `selectionguicrafting:textures/gui/background/default.png`).
+
+    ```groovy:no-line-numbers
+    background(String)
+    background(ResourceLocation)
+    ```
+
+- `ResourceLocation`. The path to the decoration texture. The resource must be loaded via external methods. [Textures that come with the mod](https://github.com/Ender-Development/selection-gui-crafting-continued/tree/master/src/main/resources/assets/selectionguicrafting/textures/gui/decor). (Default `selectionguicrafting:textures/gui/decor/default.png`).
+
+    ```groovy:no-line-numbers
+    decoration(String)
+    decoration(ResourceLocation)
+    ```
+
+- `GsEnum.OutputType`. How the output will be handed to the player. Can be overridden by the recipe. Allowed values are: `'DROP'` or `'INVENTORY'`. (Default `'DROP'`).
+
+    ```groovy:no-line-numbers
+    outputType(String)
+    outputType(GsEnum.OutputType)
+    ```
+
+- `String`. The display name of the category that will be shown in the GUI. This can either be a string or a translation key if you want to make your Display Name translatable. Requires not null.
+
+    ```groovy:no-line-numbers
+    displayName(String)
+    ```
+
+- `ResourceLocation`. The path to the progress bar texture. The resource must be loaded via external methods. Can be overridden by the recipe. [Textures that come with the mod](https://github.com/Ender-Development/selection-gui-crafting-continued/tree/master/src/main/resources/assets/selectionguicrafting/textures/gui/bar). (Default `selectionguicrafting:textures/gui/progress/default.png`).
+
+    ```groovy:no-line-numbers
+    bar(String)
+    bar(ResourceLocation)
+    ```
+
+- `GsEnum.BackgroundType`. How the background will be rendered. The resource must be loaded via external methods. Allowed values are: `'SINGLE_STRETCH'`, `'TILE'`, or `'SINGLE_CUT'`. `TILE` uses a 16x16 texture and repeats it. `SINGLE_STRETCH` stretches the texture ratio to fill the screen. `SINGLE_CUT` stretches the texture while keeping the aspect ratio and cuts of anything that goes outside the gui. The texture is centered on the x and y axis. (Default `'TILE'`).
```

Comment:
there is an inconsistency here - when listing the allowed values, they are presented with a single quote on each side `` `'SINGLE_STRETCH'` ``, but when what the value does is described, it does not have those single quotes `` `SINGLE_STRETCH` ``. should either add them or remove them from all cases. i favor removing them for a few reasons - including that this can take either an `String` or an `GsEnum.BackgroundType`, so the string part is misleading.